README.md

```diff
@@ -3,289 +3,158 @@
+> This README will focus on the individual CLI commands and how to run them.
 
-  * __init__ - Initializes a Polymer project from one of several templates
-  * __build__	- Builds an application-style project
-  * __lint__ - Lints the project
-  * __serve__	- Runs a development server
-  * __test__ - Runs tests with web-component-tester
 
 ## Installation
 
-Install via npm:
-
-    $ npm install -g polymer-cli
-
-Then run via `polymer <command>`:
-
-    $ polymer help
-
-## Project Structure
-
-Polymer-CLI is somewhat opinionated about project structure.
-
-There are two type of projects:
-
-* Elements projects
-
-  Element projects contain one or more reusable element definitions, intended to be used from other elements, applications or pages. Element definitions live at the top level of the project so they are easy to import. Elements import their dependencies with relative paths that reference sibling folders to the project folder.
-
-* Application projects
-
-  Application projects are self-contained and intended to be deployed as a standalone website. Application projects contain elements in a `src/` folder and import their dependencies with absolute paths, or relative paths that reference folders inside the project folder.
-
-### Application Styles
-
-Polymer-CLI currently supports two styles of applications:
-
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
-
-  * "App shell" applications, which have a very lightweight entrypoint, an app-shell with startup and routing logic, and possibly lazy loaded fragments.
-
-### App-shell Structure
-
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
+For best results and a faster installation, we recommend installing with [yarn](https://yarnpkg.com/en/).
 
 ## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+An application can be configured for all commands via global flags: `--entrypoint`, `--shell`, etc.
```

Comment:
This sounds pretty vague to me. An application? configured for what? etc? We haven't even mentioned the difference between applications elements by this point